.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,46 @@
+name: Local Env CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Why is it from master? Shouldn't it be from feature-branches? If it is on push to master, does it mean that tests run after we merge a new feature to master?